Octokit.Tests/Reactive/ObservableIssueCommentsClientTests.cs

```diff
@@ -89,10 +95,7 @@ public void RequestsCorrectUrlWithApiOptions()
 
                 client.GetAllForRepository("fake", "repo", options);
 
-                gitHubClient.Connection.Received(1).Get<List<IssueComment>>(
-                    new Uri("repos/fake/repo/issues/comments", UriKind.Relative), 
-                    Arg.Any<Dictionary<string, string>>(),
-                    "application/vnd.github.squirrel-girl-preview");
+                gitHubClient.Received().Issue.Comment.GetAllForRepository("fake", "repo", request, options);
```

Comment:
Id say this is the one failing, if you have a look at what the previous code was